docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,63 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Calico as a Docker network plugin service
+
+In some cases, you may want to run a calico docker image as a service.
```

Comment:
I think this could be beefed up a little to talk about running calicoctl --node --detach=false and the need to run this separately.   Basically just a little more context would be great.